### PR TITLE
intro to exceptions: test IOUs are created and exception is thrown

### PR DESCRIPTION
**Pull request by akrmn**
Co-authored-by: Gary Verhaegen <gary.verhaegen@digitalasset.com>

### CODE REVIEW

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (samuel-williams-da) [inline docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:189]: Should we have snake case here? Is this a practice I'm not aware of?

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (samuel-williams-da) [inline docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:183]: Theres `submitMultiMustFail`, so you could drop the case from the contract just exercise against the account directly?

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (garyverhaegen-da) [reply docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:189]: I'm at fault here I believe. Generally speaking we follow the Haskell convention of camelCase everywhere, but I find snake_case so much more readable that I tend to forget.

**Review by garyverhaegen-da — APPROVED** [review]
This is not a user-visible snippet so case consistency doesn't matter as much, so I'll let you decide whether you want to correct it before merging.

Either way, thanks for taking care of this!

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (akrmn) [reply docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:189]: ah yes I was going for consistency with Gary's tests. I'll switch it all to camelCase now

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (akrmn) [reply docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:183]: I thought about it but that would make a weaker test, since the exercise could fail in many different ways

Comment (samuel-williams-da) [reply docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:183]: Ah fair enough, seems we could do with some kind of assetError in script land for submitting commands

Comment (akrmn) [reply docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:183]: indeed, that would be nice to have!

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (carrielaben-da) [inline docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:163]:
```suggestion
  -- This one doesn't pass, so the balance is still 900
```

File: docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml
Comment (carrielaben-da) [inline docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml:155]:
```suggestion
  -- This transfer happened, so the balance is now 900
```

**Review by carrielaben-da — APPROVED** [review]
Just a little grammar adjustment.

**Pull request merged**
